monitor/census.go

```diff
@@ -782,8 +789,10 @@ func (cen *censusMetricsCounter) segmentEmerged(nonce, seqNo uint64, profilesNum
 }
 
 func SourceSegmentAppeared(nonce, seqNo uint64, manifestID, profile string) {
-	glog.Infof("Logging SourceSegmentAppeared... nonce=%d seqNo=%d manifestid=%s profile=%s", nonce,
-		seqNo, manifestID, profile)
+	if glog.V(logLevel) {
+		glog.Infof("Logging SourceSegmentAppeared... nonce=%d seqNo=%d manifestid=%s profile=%s", nonce,
```

Comment:
I know this was here before, but since we're in the process of cleaning this up - don't we do `manifestID`  rather than `manifestid` ? Grep is case sensitive :)

Also, I think most of our logging goes in this order `manifestID=... seqNo=...`

Reply:
changed